packages/web-components/src/components/cbp-toast/cbp-toast.tsx

```diff
@@ -0,0 +1,63 @@
+import { Component, Prop, Host, Watch, h } from '@stencil/core';
+
+@Component({
+  tag: 'cbp-toast',
+  styleUrl: 'cbp-toast.scss'
+})
+export class CbpToast {
+
+  /** specifies the icon loaded into the sidebar */
+  @Prop() icon: string = 'user';
+
+  /** specifies the color for the toast */
+  @Prop({ reflect: true }) color: 'info' | 'danger' | 'success' | 'warning' = 'info';
+
+  /** specifies time in seconds for the toast to be displayed */
+  @Prop() timer: 3 | 5 | 10;
```

Comment:
I'd call this duration - it's more accurate.

Reply:
updated name from 'timer' to 'duration'